sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -178,4 +208,19 @@ class CustomerIO internal constructor(
         event = event,
         deviceToken = deviceToken
     )
+
+    private fun recordScreenViews(activity: Activity, attributes: Map<String, Any>): Action<Unit> {
+        val packageManager = activity.packageManager
+        return try {
+            val info = packageManager.getActivityInfo(
+                activity.componentName, PackageManager.GET_META_DATA
+            )
+            val activityLabel = info.loadLabel(packageManager)
+            screen(activityLabel.toString(), attributes)
+        } catch (e: PackageManager.NameNotFoundException) {
+            ActionUtils.getErrorAction(ErrorResult(error = ErrorDetail(message = "Activity Not Found: $e")))
+        } catch (e: Exception) {
+            ActionUtils.getErrorAction(ErrorResult(error = ErrorDetail(message = "Unable to track, $activity")))
+        }
+    }
```

Comment:
Instead of this code, are we able to instead get the class name from the given Activity parameter? If an Activity class is named `LoginActivity` then we can use that class name?

Reply:
The benefit of having this is, instead of getting `class name` which doesn't necessarily depict a screen, we fetch the label given to that activity in the `Manifest`. 
A couple of clients we talked to wanted this approach as its quite standard.